mltsp/custom_feature_tools.py

```diff
@@ -466,19 +418,19 @@ def verify_new_script(script_fpath, docker_container=False):
     print(script_fpath, os.path.isfile(script_fpath))
 
     all_extracted_features = {}
-    no_docker = (os.getenv("MLTSP_NO_DOCKER") == "1")
-    if util.docker_images_available() and not no_docker:
-        print("Extracting features inside docker container...")
-        all_extracted_features = docker_extract_features(
-            script_fpath=script_fpath,
-            features_already_known=features_already_known)
-    elif no_docker:
+    if config['testing']['no_docker']:
         print("WARNING - generating custom features WITHOUT docker container...")
         all_extracted_features = execute_functions_in_order(
             features_already_known=features_already_known,
             script_fpath=script_fpath)
-    elif not util.docker_images_available():
-        raise Exception("Docker image not available.")
+    elif util.docker_images_available():
```

Comment:
This looks like the same logic as `generate_custom_features`; could these be combined?